react/src/App.jsx

```diff
@@ -0,0 +1,302 @@
+import { useState, useCallback, useEffect } from "react";
+
+function App() {
+  const [position, setPosition] = useState("bottom-right");
+
+  const [isLoggedIn, setIsLoggedIn] = useState(false);
+
+  const [selectedPrivacyOption, setSelectedPrivacyOption] = useState("disable");
+  const [selectedFullScreenOption, setSelectedFullScreenOption] =
+    useState("exitFullScreen");
+
+  const [selectedLauncherOption, setSelectedLauncherOption] =
+    useState("showLauncher");
+
+  const [unreadCount, setUnreadCount] = useState(0);
+  useState(false);
+
+  const [message, setMessage] = useState("");
+  const [messageEventListenerAdded, setmessageEventListenerAdded] =
+    useState(false);
+
+  const updateHelpshiftConfig = () => {
```

Comment:
You can just call `window.Helpshift("updateHelpshiftConfig")` wherever needed.

No need to create a new wrapper function for a one-line function call.

Reply:
Ok
- I'll fix this 
- So we have made a button for `updateHelpshiftConfig` should i remove that also 
- Currently i have done this `onClick={()=>window.Helpshift("updateHelpshiftConfig")}` on that button click instead of writing a helper function
- Is this fine or should i remove this

Reply:
`onClick={()=>window.Helpshift("updateHelpshiftConfig")}` this should be okay for this particular case.
Ideally, if it is an event, then we do create a new function to follow convention, but in this case I think it's better to have the JS API visible upfront.